README.md

```diff
@@ -1,2 +1,385 @@
-# year-cohort-projectName
-project template repo
+# Restaurant API
+
+## Resources
+MongoDB.
+
+Node.js.
+
+Express.js.
+
+RESTful API design.
+
+
+### Collection Structure
+##### Collection structure for restaurant.starters, restaurant.mains, restaurant.desserts, restaurant.refreshments
```

Comment:
Are all of these in separate collections? Would probs be better to have a single collection where each item has a category which you can use to filter them